packages/wallet-backend/src/rafiki/integration/quote/quote.service.ts

```diff
@@ -0,0 +1,64 @@
+import { BadRequestException } from '../../../shared/models/errors/BadRequestException'
+export type Amount = {
+  value: string // bigint
+  assetCode: string
+  assetScale: number
+}
+
+enum PaymentType {
+  FixedSend = 'FixedSend',
+  FixedDelivery = 'FixedDelivery'
+}
+
+export type Quote = {
+  id: string
+  paymentType: PaymentType
+  paymentPointerId: string
+  receiver: string
+  sendAmount: Amount
+  receiveAmount: Amount
+  maxPacketAmount: number // bigint
```

Comment:
Why are we not typing this as a `bigint`?

Reply:
I did, and also removed the .toString() used a bit below:
It is now just:
`receivedQuote.sendAmount.value = sendAmountValue + fees`

Reply:
And added support for bigint serialization